docs/tutorials/pytorch-mnist-tutorial.txt

```diff
@@ -156,20 +137,25 @@ The last two methods we need to define are ``build_training_data_loader`` and ``
 
 For more information on loading data in Determined, refer to the tutorial on :ref:`data-access`.
 
-Defining the Forward Pass
-"""""""""""""""""""""""""
+Defining the Training Function
+""""""""""""""""""""""""""""""
 
-The ``train_batch`` method computes the forward pass and calculates the loss for a single batch of data. Determined expects a dictionary with the calculated loss and other user-defined metrics and will automatically average all the metrics across batches. Since Determined handles the backpropagation call, we do not need to call ``loss.backwards()`` or ``optim.zero_grad()``.
+The ``train_batch`` method runs the forward passes on the losses and backward passes on the optimziers with a single batch of data. Determined expects a dictionary with the calculated loss and other user-defined metrics and will automatically average all the metrics across batches. Since ``step_optimizer`` handles the backpropagation call and zeroing out the gradients so there is no need to call ``optim.zero_grad()``.
 
 .. code:: python
 
     def train_batch(self, batch: TorchData, model: nn.Module, epoch_idx: int, batch_idx: int):
         batch = cast(Tuple[torch.Tensor, torch.Tensor], batch)
         data, labels = batch
 
+        # Define the training forward pass and calculate loss.
         output = model(data)
         loss = torch.nn.functional.nll_loss(output, labels)
 
+        # Define the training backward pass and run optimization step.
+        self.context.backward(loss)
+        self.context.step_optimizer(self.optimizer)
```

Comment:
non-blocking: split this comment into one comment per line of code here; I don't see a strong reason to fuse them in this example.

Also, instead of saying "run optimizer step", can we say "step optimizer"?